### ISSUE TITLE
automatic sleep on no movement

### ISSUE THREAD

**Issue by kolesar-andras** [zamojski/TowerCollector#6]
I use several devices for automatic data collection (see #1). It's not easy to stop recording on all devices when recording is not needed in the following situations:

* not moving for a few minutes or even hours
* being indoors
* arriving home

Sleep feature would help to:

* save battery
* save storage
* do not fill database with noise

My use case: I arrive home at late evening and my eyes are too tired to deal with a lot of phones to stop collecting. If application could go to sleep when idle I could forget all phones for night.

GPS is not suitable for movement detection as it may report false movements when signal is poor. Keeping GPS on would not help saving battery.

Accelerometer could help detecting idle time. If there is no movement for some minutes GPS may be switched off. Cell changes may be recorded with the remembered position. When accelerometer reports movement, collection should wake up and get GPS signal.

Another use case: friends could collect data in the background without any interaction without unnecessary battery drain (see also start collecting automatically on boot #1 and automatic upload #2).

**Comment by Saijin-Naib** [comment]
This is something I really thought was cool that is in Mozilla Stumbler.

I think these libraries should cover the functionality:
https://github.com/mozilla/MozStumbler/search?q=movement&unscoped_q=movement

Is it possible to add this type of functionality to Tower Collector as well?

**Comment by zamojski** [comment]
That's the long term plan :) First I need to find a way to pause Tower Collector, then automation is just a piece of cake. But that's one of the things that I'd like to have personally. Shame on me it waits for so long.

**Comment by jidanni** [comment]
Yes, the next day I'm wondering why is my phone warm to the touch? Oh, I forgot turn off Tower Collector yesterday.
Anyway  you don't even need to pause it, just kill it when the time is up, and there has been no so-called significant movement for an hour or whatever the user set it to. Another idea is to have an optional loud beeping for one whole minute before  it finally turns itself off just in case the user wants to wake it up again.

**Comment by TBNRregs** [comment]
This would be a usefull feature. So I wanted to bump it since it's been some time since last update on it.
